README.rst

```diff
@@ -566,8 +567,10 @@ Scala
 Data Science
 ~~~~~~~~~~~~
 
+* `conda-forge/cookiecutter-feedstock`_: Instruction on installing cookiecutter from conda-forge.
```

Comment:
@michaeljoseph Yeah, I realize that. I think it would be great to have install instructions on how to get  cookiecutter from the conda-forge for scientists and data scientists. perhaps docs would be a better place. 😉